### PR TITLE
chore(upgrade): update message

### PR DESCRIPTION
**Pull request by xzf0587**

### CODE REVIEW

**Comment by xzf0587** [comment]
1. align vs project version check message
2. update upgrade report for typo and old file name
3. remove the version info in error message
image

File: packages/fx-core/resource/package.nls.json [hunk 1]
```diff
@@ -129,6 +129,7 @@
   "core.projectVersionChecker.vscodeUseNewVersion": "The current project is incompatible with the installed Teams Toolkit, please install %s or above and reopen this project.",
   "core.projectVersionChecker.cliUseNewVersion": "[core] Your TeamsFx CLI is too old to support current project, please upgrade to the latest version using command below:\nnpm install -g @microsoft/teamsfx-cli@latest",
   "core.projectVersionChecker.incompatibleProject": "The current project is incompatible with the installed version of Teams Toolkit.",
+  "core.projectVersionChecker.vs.incompatibleProject": "The project in the solution is created with Teams Toolkit preview feature - Teams App Configuration Improvements. You can turn on the preview feature to continue.",
```
Comment (xzf0587) [inline packages/fx-core/resource/package.nls.json:132]: This message is used in vs already. Update here to keep aligned

File: packages/fx-core/resource/package.nls.json [hunk 2]
```diff
@@ -372,7 +373,7 @@
   "error.DotEnvFileNotExistError": "Environment %s not found. Make sure the .env.%s file exist.",
   "error.InvalidEnvNameError": "Environment name can only contain letters, digits, _ and -.",
   "error.UpgradeCanceledError": "If you don't want to upgrade your project, install another version of Teams Toolkit (version <= 2.10.0).",
-  "error.UpgradeV3CanceledError": "If you don't want to upgrade your project, install another version of Teams Toolkit in %s based on the project version %s.",
+  "error.UpgradeV3CanceledError": "If you are not ready to upgrade, please continue to use the old version Teams Toolkit",
```
Comment (sffamily) [inline packages/fx-core/resource/package.nls.json:376]: old version -> old version of

**Review by blackchoey — APPROVED** [review]

File: packages/fx-core/resource/package.nls.json (hunk 2 above)
Comment (xzf0587) [reply packages/fx-core/resource/package.nls.json]: Fixed. Thanks.

File: packages/fx-core/resource/package.nls.json (hunk 1 above)
Comment (jayzhang) [reply packages/fx-core/resource/package.nls.json:132]: can you follow the key pattern rule: error.migration.{sub-category}.{Error name capitalized first letter}

Comment (xzf0587) [reply packages/fx-core/resource/package.nls.json:132]: Here is a warning message, not error message. Will follow this format in new error message

**Review by jayzhang — APPROVED** [review]

**Review by sffamily — APPROVED** [review]

**Pull request merged**